src/main/java/com/celements/cells/CellRenderStrategy.java

```diff
@@ -58,7 +58,7 @@
 
 public class CellRenderStrategy implements IRenderStrategy {
 
-  public static final String EXEC_CTX_KEY = "celements.globalvalues.cell";
+  public static final String EXEC_CTX_KEY = "celements.cell";
```

Comment:
is it correct to remove "globalvalues." from the constant if considering the same constant in CelementsWebScriptService.java:102 still showing "globalvalues."?

Reply:
I've removed `globalvalues` since I don't want to expose the rendering scope to velocity, mostly because we're now adding XWikiDocument instances to it.
In order to not break legacy solutions (which indeed manipulate `celements.globalvalues.cell.number` from velocity) I've added a fallback for it where necessary.

Reply:
👍